privly.safariextension/Info.plist

```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
```

Comment:
The privly safari extension version has been changed to 0.1.0 under `CFBundleShortVersionString`